Makefile

```diff
@@ -252,7 +282,55 @@ start-iso-abi: checkenv bip-orchestrate-vm
 		RELEASE_VERSION=$(RELEASE_VERSION) \
 		CPU_CORE=$(CPU_CORE) \
 		RELEASE_ARCH=$(RELEASE_ARCH) \
-		RAM_MB=$(RAM_MB)
+		RAM_MB=$(RAM_MB) \
+		BASE_DOMAIN=$(BASE_DOMAIN) \
+		NET_NAME=$(NET_NAME) \
+		NET_BRIDGE_NAME=$(NET_BRIDGE_NAME) \
+		NET_UUID=$(NET_UUID) \
+		NET_MAC=$(NET_MAC)
+
+# Network 0 is used for the seed VM
+network_0: BASE_DOMAIN=$(NET_0_DOMAIN)
```

Comment:
Consider renaming to seed_network and recipient_network (it's OK if IBI will use the recipient network)